app/logic/volunteers.py

```diff
@@ -92,3 +98,26 @@ def setUserBackgroundCheck(user,bgType, checkPassed, datePassed):
     update = BackgroundCheck.create(user=user, type=bgType, passBackgroundCheck=checkPassed, datePassed=datePassed)
     update.save()
     createLog(f"Updated {user.firstName} {user.lastName}'s background check for {bgType} to {bool(checkPassed)}.")
+
+def isProgramManagerForEvent(user, event= None):
+    """
+    This function checks to see if a user is a program manager for a program.
+    NOTE: this function needs the event parameter to work
+    user: expects a user peewee object
+    event: expects an event peewee object
+    Returns: bool whether the appropriate information is given.
+    """
+
+    #neither event nor programId are passed
+    if not (event):
+        raise ValueError("Not enough parameters given to this function.")
+
+    #if event is passed but no programId is passed
+    if event:
+        programIdQuery = (ProgramEvent.get(ProgramEvent.event == event)).program
+        programManagerResult = hasPrivilege(user, programIdQuery)
+    if programManagerResult == True:
+        return True
+    else:
+        # print(e)
+        return False
```

Comment:
Instead of saying essentially "if true equals true return true", just return the result. It's already True or False.